lib/gis/parser_rest_md.c

```diff
@@ -51,7 +53,15 @@ void usage_rest_md(bool rest)
     if (!st->pgm_name)
         st->pgm_name = "??";
 
-    /* main header */
+    /* print metadata used by man/build*.py */
+    fprintf(stdout, "---\n");
+    fprintf(stdout, "module: %s\n", st->pgm_name);
```

Comment:
Detail, but perhaps we want this to be simply name (of the module/tool)?

```suggestion
    fprintf(stdout, "name: %s\n", st->pgm_name);
```